lib/fake_braspag/views/sales_cancel.jbuilder

```diff
@@ -0,0 +1,6 @@
+json.Status 10
```

Comment:
Voided with successful, this template is a mock from a successful cancellation response.